### ISSUE TITLE
Contact added Birthday's doesn't show on Calendar

### ISSUE THREAD

**Issue by Tetroner9** [8bitPit/Niagara-Issues#3053]
### Describe the bug

Birthdays which are added directly to someone's name doesn't show up on the home screen as the upcoming event or even after opening the whole bar. Opening the Google Calendar shows the birthdays. I have set up the filters correctly both on the launcher and calendar 

### Steps to reproduce

1. Open the calendar widget 
2. Expand the widget

### Does the issue also occur with other 3rd-party launchers?

Not applicable

### Device name

One Plus Nord CE 2 Lite 5G

### Android version

Android 14

**Comment by TFackler** [comment]
Thanks for reporting this issue.

Could you please verify whether the steps listed here fix the issue: [How can I show my contacts' birthdays](https://help.niagaralauncher.app/article/76-calendar-faq#:~:text=How%20can%20I%20show%20my%20contacts%27%20birthdays%3F).

**Comment by Tetroner9** [comment]
Yes, enabling and disabling the birthdays filter and refreshing fixed the issue. Thank you!
